### PR TITLE
Add Vue3 as front part in JHipster Lite

### PR DESCRIPTION
**Pull request by pascalgrimaud**
This is a first draft to add Vue3 as front part in JHipster Lite. How I did this:
- generate a new project with vue-cli + Vue3
- change config and folder to use src/main/webapp/app
- copy and merge folders/config it in JHipster Lite
- add build part in pom.xml

This ticket is:
- related to https://github.com/jhipster/jhipster-lite/issues/295

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,6 +97,10 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-validation</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-thymeleaf</artifactId>
+    </dependency>
```
Comment (pblanchardie) [inline pom.xml:103]: I'm curious, what is the purpose of Thymeleaf in this PR?

Comment (pascalgrimaud) [reply pom.xml:103]: As Thymeleaf is a template engine, used for displaying html, js, etc, I think it is needed but I doubt now :-D
I need to test without it

Comment (pblanchardie) [reply pom.xml:103]: For server side rendering only (like mustache, for xml)

Comment (pascalgrimaud) [reply pom.xml]: you're right, it's not needed. Removed :)
Thanks

**Comment by pascalgrimaud** [comment]
closing in favor of https://github.com/jhipster/jhipster-lite/pull/668

**Pull request closed without merging**